app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -83,6 +95,31 @@
   color: #CDCDCD;
 }
 
+.ModuleSearch .Input .ClearButton {
+  --iconSize: 14px;
+  --iconPadding: 3px; /* give additional hover leeway */
+
+  width: calc(2 * var(--iconPadding) + var(--iconSize));
```

Comment:
To make it more friendly we could
```css
.ModuleSearch .Input .ClearButton {
  box-sizing: border-box;
  padding: 3px;
  width: 20px;
  height: 20px;
  /* … */
}
```
I kinda is self-explanatory then.

Reply:
you mean `content-box`, right? With `content-box` we don't need to do the `width = width + padding` math.

i.e. 

```css
…
  box-sizing: content-box;
  width: 14px; /* Actual rendered size of icon */
  height: 14px; 
  padding: 3px; /* Additional space on top */
…
```